Cargo.toml

```diff
@@ -1,5 +1,5 @@
 [workspace]
 
 members = [
-    "crd", "operator", "server"
+    "crd", "operator", "server", "common"
```

Comment:
Should the test_utils stuff also be here?

Reply:
I was not sure about that since its not flagged with #[cfg(test)]. I did not want it in the official workspace. Ill add it for now.